protocols/cosi/cosi.go

```diff
@@ -86,14 +90,6 @@ func NewProtocolCosi(node *sda.Node) (*ProtocolCosi, error) {
 	return pc, err
 }
 
-// NewRootProtocolCosi is used by the root to collectively sign this message
-// (vanilla version of the protocol where no contributions are done)
-func NewRootProtocolCosi(msg []byte, node *sda.Node) (*ProtocolCosi, error) {
-	pc, err := NewProtocolCosi(node)
-	pc.message = msg
-	return pc, err
-}
```

Comment:
I am not so sure about removing that: Creating a RootProtocolCoSi means that this node is the *leader*, so it, and only it, knows about the message to sign yet. In the test you give it to every node on the creation, it feels wrong since the message is actually to be passed in the first message, why would you want to cut that out ?